src/qt/clientmodel.cpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2011-2020 The Bitcoin Core developers
+// Copyright (c) 2011-2019 The Bitcoin Core developers
```

Comment:
In commit "Added best block hash to the NotifyHeaderTip and NotifyBlockTip signals." (6002d1185db89af124a257c894195a7f1b04c1f2)

Probably unintended change this line

Reply:
yeah, rebase issue. Good catch.